.vscode/tasks.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
This task was necessary to make sure that debug launches in vscode did a build first.